Makefile

```diff
@@ -52,17 +53,19 @@ _output/share/lima/lima-guestagent.Linux-aarch64:
 
 .PHONY: install
 install:
-	cp -av _output/* "$(DESTDIR)/$(PREFIX)/"
-	if [[ $(shell uname -s ) != Linux && ! -e "$(DESTDIR)/$(PREFIX)/bin/nerdctl" ]]; then ln -sf nerdctl.lima "$(DESTDIR)/$(PREFIX)/bin/nerdctl"; fi
+	mkdir -p $(DEST)
```

Comment:
Every expression that interpolates `$(DEST)` (or any other variable) needs to be put in quotes. Otherwise it will break when the path contains spaces. This applies also to virtually all new lines below (see the original lines, that did include the quotes; not sure why you removed them).

Reply:
I did it because of:

```console
$ DESTDIR=/tmp/ PREFIX="/foo bar  baz" make install
mkdir -p "/tmp/foo bar  baz"
cp -av _output/* "/tmp/foo bar  baz"
_output/bin -> /tmp/foo bar  baz/bin
_output/bin/lima -> /tmp/foo bar  baz/bin/lima
_output/bin/nerdctl.lima -> /tmp/foo bar  baz/bin/nerdctl.lima
_output/bin/limactl -> /tmp/foo bar  baz/bin/limactl
_output/share -> /tmp/foo bar  baz/share
_output/share/lima -> /tmp/foo bar  baz/share/lima
_output/share/lima/lima-guestagent.Linux-x86_64 -> /tmp/foo bar  baz/share/lima/lima-guestagent.Linux-x86_64
_output/share/lima/lima-guestagent.Linux-aarch64 -> /tmp/foo bar  baz/share/lima/lima-guestagent.Linux-aarch64
_output/share/doc -> /tmp/foo bar  baz/share/doc
_output/share/doc/lima -> /tmp/foo bar  baz/share/doc/lima
_output/share/doc/lima/LICENSE -> /tmp/foo bar  baz/share/doc/lima/LICENSE
_output/share/doc/lima/docs -> /tmp/foo bar  baz/share/doc/lima/docs
_output/share/doc/lima/docs/internal.md -> /tmp/foo bar  baz/share/doc/lima/docs/internal.md
_output/share/doc/lima/README.md -> /tmp/foo bar  baz/share/doc/lima/README.md
_output/share/doc/lima/VERSION -> /tmp/foo bar  baz/share/doc/lima/VERSION
_output/share/doc/lima/examples -> /tmp/foo bar  baz/share/doc/lima/examples
_output/share/doc/lima/examples/debian.yaml -> /tmp/foo bar  baz/share/doc/lima/examples/debian.yaml
_output/share/doc/lima/examples/alpine.yaml -> /tmp/foo bar  baz/share/doc/lima/examples/alpine.yaml
_output/share/doc/lima/examples/ubuntu.yaml -> /tmp/foo bar  baz/share/doc/lima/examples/ubuntu.yaml
_output/share/doc/lima/examples/fedora.yaml -> /tmp/foo bar  baz/share/doc/lima/examples/fedora.yaml
_output/share/doc/lima/examples/README.md -> /tmp/foo bar  baz/share/doc/lima/examples/README.md
_output/share/doc/lima/examples/archlinux.yaml -> /tmp/foo bar  baz/share/doc/lima/examples/archlinux.yaml
_output/share/doc/lima/examples/opensuse.yaml -> /tmp/foo bar  baz/share/doc/lima/examples/opensuse.yaml
_output/share/doc/lima/examples/k3s.yaml -> /tmp/foo bar  baz/share/doc/lima/examples/k3s.yaml
_output/share/doc/lima/examples/k3s-rootless.yaml -> /tmp/foo bar  baz/share/doc/lima/examples/k3s-rootless.yaml
_output/share/doc/lima/examples/default.yaml -> /tmp/foo bar  baz/share/doc/lima/examples/default.yaml
if [[ Darwin != Linux && ! -e "/tmp/foo bar  baz"/bin/nerdctl ]]; then ln -sf nerdctl.lima "/tmp/foo bar  baz"/bin/nerdctl; fi
```

```console
DESTDIR=/tmp/ PREFIX="/foo bar  baz" make uninstall
rm -rf \
	"/tmp/foo bar  baz"/bin/lima \
	"/tmp/foo bar  baz"/bin/limactl \
	"/tmp/foo bar  baz"/bin/nerdctl.lima \
	"/tmp/foo bar  baz"/share/lima "/tmp/foo bar  baz"/share/doc/lima
```

While, with quotes:

```diff
diff --git a/Makefile b/Makefile
index 3e3dd4b..c47c5b0 100644
--- a/Makefile
+++ b/Makefile
@@ -53,18 +53,18 @@ _output/share/lima/lima-guestagent.Linux-aarch64:
 
 .PHONY: install
 install:
-       mkdir -p $(DEST)
-       cp -av _output/* $(DEST)
-       if [[ $(shell uname -s ) != Linux && ! -e $(DEST)/bin/nerdctl ]]; then ln -sf nerdctl.lima $(DEST)/bin/nerdctl; fi
+       mkdir -p "$(DEST)"
+       cp -av _output/* "$(DEST)"
+       if [[ $(shell uname -s ) != Linux && ! -e "$(DEST)/bin/nerdctl" ]]; then ln -sf nerdctl.lima "$(DEST)/bin/nerdctl"; fi
 
 .PHONY: uninstall
 uninstall:
-       @test -f $(DEST)/bin/lima || (echo "lima not found in $(DEST) prefix"; exit 1)
+       @test -f "$(DEST)/bin/lima" || (echo "lima not found in $(DEST) prefix"; exit 1)
        rm -rf \
-               $(DEST)/bin/lima \
-               $(DEST)/bin/limactl \
-               $(DEST)/bin/nerdctl.lima \
-               $(DEST)/share/lima $(DEST)/share/doc/lima
+               "$(DEST)/bin/lima" \
+               "$(DEST)/bin/limactl" \
+               "$(DEST)/bin/nerdctl.lima" \
+               "$(DEST)/share/lima $(DEST)/share/doc/lima"
        # TODO: remove $(DEST)/bin/nerdctl only when it is a symlink to nerdctl.lima
```

```console
DESTDIR=/tmp/ PREFIX="/foo bar  baz" make install
mkdir -p ""/tmp/foo bar  baz""
cp -av _output/* ""/tmp/foo bar  baz""
_output/bin -> baz/bin
_output/bin/lima -> baz/bin/lima
_output/bin/nerdctl.lima -> baz/bin/nerdctl.lima
_output/bin/limactl -> baz/bin/limactl
_output/share -> baz/share
_output/share/lima -> baz/share/lima
_output/share/lima/lima-guestagent.Linux-x86_64 -> baz/share/lima/lima-guestagent.Linux-x86_64
_output/share/lima/lima-guestagent.Linux-aarch64 -> baz/share/lima/lima-guestagent.Linux-aarch64
_output/share/doc -> baz/share/doc
_output/share/doc/lima -> baz/share/doc/lima
_output/share/doc/lima/LICENSE -> baz/share/doc/lima/LICENSE
_output/share/doc/lima/docs -> baz/share/doc/lima/docs
_output/share/doc/lima/docs/internal.md -> baz/share/doc/lima/docs/internal.md
_output/share/doc/lima/README.md -> baz/share/doc/lima/README.md
_output/share/doc/lima/VERSION -> baz/share/doc/lima/VERSION
_output/share/doc/lima/examples -> baz/share/doc/lima/examples
_output/share/doc/lima/examples/debian.yaml -> baz/share/doc/lima/examples/debian.yaml
_output/share/doc/lima/examples/alpine.yaml -> baz/share/doc/lima/examples/alpine.yaml
_output/share/doc/lima/examples/ubuntu.yaml -> baz/share/doc/lima/examples/ubuntu.yaml
_output/share/doc/lima/examples/fedora.yaml -> baz/share/doc/lima/examples/fedora.yaml
_output/share/doc/lima/examples/README.md -> baz/share/doc/lima/examples/README.md
_output/share/doc/lima/examples/archlinux.yaml -> baz/share/doc/lima/examples/archlinux.yaml
_output/share/doc/lima/examples/opensuse.yaml -> baz/share/doc/lima/examples/opensuse.yaml
_output/share/doc/lima/examples/k3s.yaml -> baz/share/doc/lima/examples/k3s.yaml
_output/share/doc/lima/examples/k3s-rootless.yaml -> baz/share/doc/lima/examples/k3s-rootless.yaml
_output/share/doc/lima/examples/default.yaml -> baz/share/doc/lima/examples/default.yaml
/tmp/foo -> baz/foo
bar -> baz/bar
if [[ Darwin != Linux && ! -e ""/tmp/foo bar  baz"/bin/nerdctl" ]]; then ln -sf nerdctl.lima ""/tmp/foo bar  baz"/bin/nerdctl"; fi
/bin/sh: -c: line 0: syntax error in conditional expression
/bin/sh: -c: line 0: syntax error near `bar'
/bin/sh: -c: line 0: `if [[ Darwin != Linux && ! -e ""/tmp/foo bar  baz"/bin/nerdctl" ]]; then ln -sf nerdctl.lima ""/tmp/foo bar  baz"/bin/nerdctl"; fi'
make: *** [Makefile:58: install] Error 2
```

And assuming it was installed at the step before...

```console
DESTDIR=/tmp/ PREFIX="/foo bar  baz" make uninstall
/bin/sh: line 0: test: too many arguments
lima not found in /tmp/foo bar baz prefix
make: *** [Makefile:62: uninstall] Error 1
```

---

```console
make --version
GNU Make 4.3
```

Reply:
You are absolutely right, and it is actually my fault for (incorrectly) insisting that

```
DEST := "$(shell echo "$(DESTDIR)/$(PREFIX)" | sed 's:///*:/:g; s://*$$::')"
```

must include quotes on the right hand side (because I was testing with `@echo $(DEST)` instead of `@echo "$(DEST)"`). That was wrong; the quotes should not be part of the value, but should be added when the variable is used.

So please remove the quotes from the `DEST := ...` line and restore them in all the original places!

Sorry for causing this confusion and extra work!